components/Wizard/ImageStep.js

```diff
@@ -292,20 +291,141 @@ class ImageStep extends React.PureComponent {
+            : formatMessage(messages.warningSizeSmall, {
+                size: this.props.minImageSize,
+              })
+        }
+        helperTextIcon={
+          imageSizeValidated === "warning" ? <ExclamationTriangleIcon className="cc-c-text__align-icon" /> : ""
+        }
+        helperTextInvalid={imageSizeHelperTextInvalid}
+        isRequired
+        validated={imageSizeValidated}
+      >
+        <InputGroup>
+          <TextInput
+            className="pf-c-form-control"
+            id="image-size-input"
+            aria-describedby="image-size-input-helper"
+            type="number"
+            min={minImageSize}
+            max={maxImageSize}
+            value={imageSize || ""}
+            validated={imageSizeValidated}
+            onChange={this.handleImageSizeInput}
+            isRequired
+          />
+          <InputGroupText>GB</InputGroupText>
+        </InputGroup>
+      </FormGroup>
+    );
+
+    const ostreeFields = (
+      <>
+        <FormGroup
+          label={<FormattedMessage defaultMessage="Parent commit" />}
+          labelIcon={
+            <Popover
+              id="ostree-parent-popover"
+              bodyContent={
+                <FormattedMessage defaultMessage="Provide the ID of the latest commit in the updates repository for which this commit provides an update." />
+              }
+              aria-label={formatMessage(ariaLabels.ostreeParent)}
+            >
+              <Button variant="plain" aria-label={formatMessage(ariaLabels.ostreeParent)}>
+                <OutlinedQuestionCircleIcon className="cc-c-text__align-icon" id="popover-icon" />
+              </Button>
+            </Popover>
+          }
+          fieldId="ostree-parent-input"
+          hasNoPaddingTop
+        >
+          <TextInput
+            className="pf-c-form-control"
+            value={ostreeSettings.parent !== undefined ? ostreeSettings.parent : ""}
+            type="text"
+            id="ostree-parent-input"
+            onChange={setOstreeParent}
+          />
+        </FormGroup>
+        <FormGroup
+          label={<FormattedMessage defaultMessage="Ref" />}
+          labelIcon={
+            <Popover
+              id="ostree-ref-popover"
+              bodyContent={
+                <FormattedMessage defaultMessage="Provide the name of the branch for the content. If the ref does not already exist it will be created." />
+              }
+              aria-label={formatMessage(ariaLabels.ostreeRef)}
+            >
+              <Button variant="plain" aria-label={formatMessage(ariaLabels.ostreeRef)}>
+                <OutlinedQuestionCircleIcon className="cc-c-text__align-icon" id="popover-icon" />
+              </Button>
+            </Popover>
+          }
+          fieldId="ostree-ref-input"
+          helperText={formatMessage(messages.ostreeRefHelperText)}
+          helperTextInvalid={formatMessage(messages.ostreeRefHelperText)}
```

Comment:
I'm not seeing this helper text display, and I'm not sure why. Are you seeing it in your build?

<img width="814" alt="image" src="https://user-images.githubusercontent.com/21063328/99122143-cb0abf00-25cb-11eb-99ee-d542ac1dd5c0.png">

Reply:
I'm deleting this comment, because I think I realized what the issue was on my end. 🙄